From 2837f02329ccec4b6675a8fe39bcab19739203ba Mon Sep 17 00:00:00 2001
From: Yannick Lanz <yannick.lanz@wifx.net>
Date: Wed, 17 Apr 2019 10:15:01 +0200
Subject: [PATCH 1/2] mk: break up long SED_REPLACE line

No functional changes, just breaking up a sed command long line similar
to src/librc/Makefile.
---
 mk/scripts.mk | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/mk/scripts.mk b/mk/scripts.mk
index 147e6643..81cb1e73 100644
--- a/mk/scripts.mk
+++ b/mk/scripts.mk
@@ -19,7 +19,15 @@ _PKG_SED:=		$(shell ${_PKG_SED_SH})
 _LCL_SED_SH=		if test "${PREFIX}" = "${LOCAL_PREFIX}"; then echo "-e 's:@LOCAL_PREFIX@::g'"; else echo "-e 's:@LOCAL_PREFIX@:${LOCAL_PREFIX}:g'"; fi
 _LCL_SED:=		$(shell ${_LCL_SED_SH})
 
-SED_REPLACE=		-e 's:@SHELL@:${SH}:' -e 's:@LIB@:${LIBNAME}:g' -e 's:@SYSCONFDIR@:${SYSCONFDIR}:g' -e 's:@LIBEXECDIR@:${LIBEXECDIR}:g' -e 's:@PREFIX@:${PREFIX}:g' -e 's:@BINDIR@:${BINDIR}:g' -e 's:@SBINDIR@:${SBINDIR}:g' ${_PKG_SED} ${_LCL_SED}
+SED_REPLACE=	-e 's:@SHELL@:${SH}:'
+SED_REPLACE+=	-e 's:@LIB@:${LIBNAME}:g'
+SED_REPLACE+=	-e 's:@SYSCONFDIR@:${SYSCONFDIR}:g'
+SED_REPLACE+=	-e 's:@LIBEXECDIR@:${LIBEXECDIR}:g'
+SED_REPLACE+=	-e 's:@PREFIX@:${PREFIX}:g'
+SED_REPLACE+=	-e 's:@BINDIR@:${BINDIR}:g'
+SED_REPLACE+=	-e 's:@SBINDIR@:${SBINDIR}:g'
+SED_REPLACE+=	${_PKG_SED}
+SED_REPLACE+=	${_LCL_SED}
 
 # Tweak our shell scripts
 %.sh: %.sh.in
-- 
2.17.1

